Formula/smake.rb

```diff
@@ -17,11 +18,8 @@ class Smake < Formula
   end
 
   def install
-    # The bootstrap smake does not like -j
-    ENV.deparallelize
-    # Xcode 9 miscompiles smake if optimization is enabled
-    # https://sourceforge.net/p/schilytools/tickets/2/
-    ENV.O1 if DevelopmentTools.clang_build_version >= 900
+    # ARM fails to build with Os
+    ENV.O1 if Hardware::CPU.arm?
```

Comment:
This should probably be reported somewhere though I don't know where.

Reply:
It was here: https://sourceforge.net/p/schilytools/tickets/2/

It's fixed in smake 1.3 (I think), which is not independently packaged anymore. It's in the latest `schilytools`, though that updates more often than `smake` gets a version bump so you'll need to inspect the source files for the actual version number (and even then there may be stealth changes in newer archives).